src/tsconfig.json

```diff
@@ -17,22 +17,21 @@
         "noImplicitAny": false,
         "noImplicitReturns": true,
         "noImplicitUseStrict": false,
-        "noFallthroughCasesInSwitch": true
+        "noFallthroughCasesInSwitch": true,
+        "skipLibCheck": true
     },
     "filesGlob": [
         "*.ts"
     ],
     "include": [
+        "./node_modules/tns-platform-declarations/android.d.ts",
+        "./nativescript-fresco.android-map.d.ts",
         "./nativescript-fresco-common.ts",
         "./nativescript-fresco.android.ts",
         "./nativescript-fresco.ios.ts",
         "./angular/nativescript-fresco-directives.ts",
         "./angular/nativescript-fresco.module.ts",
-        "./angular/index.ts",
-        "references.d.ts"
-    ],
-    "exclude": [
-        "node_modules"
```

Comment:
Why did you remove this exclude? Running tsc on the node_modules should not be necessary. Multiple plugins including the [plugin seed](https://github.com/NativeScript/nativescript-plugin-seed/blob/master/src/tsconfig.json) add the node_modules to the excludes of the tsconfig.

Reply:
@VladimirAmiorkov 
When using `include` it isn't necessary to exclude `node_modules`, since it isn't in the `include`, so therefore won't be included. The only time you'd want to use include and exclude was if your include had something like `libs/**` and then your exclude had `libs/xyz` because maybe you have lots of stuff in `libs` that you don't want to list out in the `include` but you want `libs/xyz` to be excluded for some reason.

I know `plugin seed` and many others use `exclude`, but usually that's just because they have no `include`, so the default behaviour is to include everything within the directory so you just filter out `node_modules`, but if you're already using `include` then excluding `node_modules` is redundant and can be misleading.

For ease of use I think `plugin seed` and many others just use `exclude` because then you don't need to think about what files need to be compiled, but if you have `include` you already have, and `include` is much more powerful and scoped and less likely to result in strange behaviour in the long run.

*tl;dr*: because `include` is already being used and `node_modules` isn't in there, it is already being excluded from the `tsc`.

Reply:
Yeah, I removed the includes in the new PR that will bring the latest native lib to the plugin for this reason. It looks like when this plugin was developer back in the day the decision to use include was taken but I don't see ant reason to use it rather than use the exclude option.